scripts/common/prompts.sh

```diff
@@ -200,6 +200,8 @@ promptForLoadBalancerAddress() {
     if [ -z "$LOAD_BALANCER_PORT" ]; then
         LOAD_BALANCER_PORT=6443
     fi
+
+    $BIN_BASHTOYAML -c $MERGED_YAML_SPEC -f "load-balancer-address=${LOAD_BALANCER_ADDRESS}:${LOAD_BALANCER_PORT}"
```

Comment:
This means the merged yaml spec will always have a load balancer address and the tcp load balancer check will always run, even on non-ha clusters, right?

Reply:
we only prompt for a load balancer in ha clusters

https://github.com/replicatedhq/kURL/blob/28855945e6db7addf65fe5eeab47899fe3c102d5/scripts/common/prompts.sh#L12-L14

```bash
    if [ "$HA_CLUSTER" = "1" ]; then
        promptForLoadBalancerAddress
    fi
```

Reply:
To clarify, we template the preflight spec with the merged yaml spec. If we dont merge in values from cli flags and prompts then its not possible to base conditional checks based on user input.